bin/node-template/pallets/template/src/lib.rs

```diff
@@ -75,7 +75,7 @@ decl_module! {
 		/// Just a dummy entry point.
 		/// function that can be called by the external world as an extrinsics call
 		/// takes a parameter of the type `AccountId`, stores it, and emits an event
-		#[weight = 0]
+		#[weight = 10_000]
```

Comment:
I'm not super familiar with this but Shawn could suggest another way to calculate a simple default value.
Something like:  `(10_000 + T::DbWeight::get().writes(1))]`